docs/src/pages/company/careers/product-designer.md

```diff
@@ -1,92 +1,74 @@
-# Product Designer
+# Designer
 
 <p class="description">Design is critical to the success of our mission. We are looking for skills that complement our Lead Designer. You will empower our audience that seeks to build outstanding-looking UIs with new tools.</p>
 
-## Details of the Role
+## About the company
 
-- Location: Remote (preference for UTC-6 to UTC+3).
-- Type of work: Full-time (contractor or employee [depending on the case](https://www.notion.so/mui-org/Hiring-FAQ-64763b756ae44c37b47b081f98915501))
-- Start date: Immediately.
-- We're a remote company, we prefer asynchronous communication over meetings.
-- We work independently, the rest of us won't know what you're doing day-to-day unless you tell us.
+MUI started with Material UI, the most successful React implementation of Google’s Material Design. It has gotten a massive following not only due to the fidelity to Material but also because its number of components, carefully designed component API, obsession for details, and community engagement. Today, countless teams and organizations rely on our open-source projects to build their design system.
 
-## About us
+A few years back, we started expanding our suite of products. We released [MUI X](/x), a collection of advanced components, [MUI Design Kits](/design-kits), the MUI Material components available in the most popular design tools out there, and [MUI Templates](/templates), a set of pre-built UIs. We’ve seen fast growth in all of them (x2 ~ 3 year over year) and we have much more baking in the oven.
 
-See the [careers](/careers/), [handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b), and [about us](/about/) pages.
+We are a team of fully distributed individuals, spread across multiple timezones. We work mainly asynchronously, relying on lots of documentation and written collaboration. Every contributor has the freedom to define how they work — the rest of us won’t know what you’re doing day-to-day unless you tell us. For additional details about MUI team and culture, you can check our [careers](/careers) and [about](/about) pages and also our [public Handbook](https://www.notion.so/Handbook-f086d47e10794d5e839aef9dc67f324b).
 
-### Why we're hiring
+## Why we’re hiring
```

Comment:
The current policy is to only use straight quotes. It's coming from a discussion with Blake Ramick, a designer, a year ago. To avoid inconsistancies, we normalized it all.

```suggestion
## Why we're hiring
```

*x* the other instances in this document.